src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -853,7 +853,8 @@ public function bulkAction(Request $request, $action)
                         $hookEventParameters
                     );
                     // Hooks: managed in ProductUpdater
-                    $productUpdater->duplicateProductIdList($productIdList);
+                    $duplicateProducstId = $productUpdater->duplicateProductIdList($productIdList);
```

Comment:
```suggestion
                    $productUpdater->duplicateProductIdList($productIdList, $duplicateProducstId);
```

Maybe you can use an optional parameter to perform the action without breaking the interface?